### DISCUSSION TITLE
Unable to add my chatwoot DOMAIN on Insta/Face configuration

### DISCUSSION THREAD

**Discussion by pcajubr** [chatwoot/chatwoot#4725] [Self hosted] [answered]
Hi, I have chatwoot self-hosted (v.2.5.0)

On Instagram and Facebook app configuration (on Meta Developers), dont work the step that ADD CHATWOOT DOMAIN

![image](https://user-images.githubusercontent.com/106082805/169821303-60a8bdcf-391b-466d-9c6e-d3045f5fa57d.png)

Notihing happens. Clicking F12 show me error:  Async request failed with error 1007

**Comment by tejaswinichile** [comment, marked as answer]
@pcajubr Hello, we don't do any changes in the advanced settings.
You need to add your chatwoot domain in the basic settings.

https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/instagram-channel-setup#configure-the-facebook-app
here.
<img width="600" alt="basic_settings" src="https://user-images.githubusercontent.com/10827585/169837397-35c2162a-11fb-4b99-afb8-f36697120721.png">


Just set the version in advanced settings and allow API access.

<img width="600" alt="advanced_setting" src="https://user-images.githubusercontent.com/10827585/169837682-c00bf4e5-5f9e-4e29-8bf5-d5289ff4d8c7.png">
